src/service/VotingUtils.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2021 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { existsSync, lstatSync, readdirSync, readFileSync } from 'fs';
+import * as noble from 'noble-ed25519';
+import * as forge from 'node-forge';
+import { join } from 'path';
+import { Convert, Crypto } from 'symbol-sdk';
+import * as nacl from 'tweetnacl';
+
+const ed25519 = forge.pki.ed25519;
+export interface KeyPair {
+    privateKey: Uint8Array;
+    publicKey: Uint8Array;
+}
+export interface CryptoImplementation {
+    name: string;
```

Comment:
Nice.  Good TS impl.

I think this file should be moved to a lib which can get used by Wallet or Symbol-cli.